### PR TITLE
P2022-502 Github Action Continuous Delivery -> P2022-697 release apk workflow

### PR DESCRIPTION
**Pull request by BorysChajdas**

### CODE REVIEW

File: .github/workflows/build-and-upload-apk.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ on:
...
     inputs:
       release_name:
-        required: true
+        required: false
         description: "Release name"
 
 jobs:
-  assembleProjectAndCreateRelease:
+  BuildAndRelease:
     runs-on: ubuntu-latest
```
Comment (MarekPawlowskiAuto) [inline .github/workflows/build-and-upload-apk.yml:11]: If we choose latest, we will not have control when github actions will update ubuntu(and in result our CI/CD), currently its "ubuntu-20.04", i would  still change latest to ubuntu-20.04, in this case we will not have some surprises if something change. Possibility of it is super rare, plus this is only place where container is running, but still....

Comment (MarekPawlowskiQA) [inline .github/workflows/build-and-upload-apk.yml:11]: If we choose latest, we will not have control when github actions will update ubuntu(and in result our CI/CD), currently its "ubuntu-20.04", i would still change latest to ubuntu-20.04, in this case we will not have some surprises if something change. Possibility of it is super rare, plus this is only place where container is running, but still....

Comment (BorysChajdas) [reply .github/workflows/build-and-upload-apk.yml]: Yeah, I agree with you. Changed to version 20.04 👍 .

**Review by MarekPawlowskiAuto — APPROVED** [review]
lgtm :)

**Review by MarekMacko — APPROVED** [review]

**Pull request merged**